web/viewer.css

```diff
@@ -1703,6 +1703,22 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
   .secondaryToolbarButton.documentProperties::before {
     content: url(images/secondaryToolbarButton-documentProperties@2x.png);
   }
+
+  .outlineItemToggler::before {
+    -webkit-transform: scale(0.5);
+    transform: scale(0.5);
+    top: -1px;
+    content: url(images/treeitem-collapsed@2x.png);
+  }
+  .outlineItemToggler.outlineItemsHidden::before {
+    content: url(images/treeitem-expanded@2x.png);
+  }
+  html[dir="ltr"] .outlineItemToggler::before {
```

Comment:
Use single quotes instead of double quotes (same for below), because single quotes are much more common:

```
web$ grep \" viewer.css | wc -l
21
web$ grep \' viewer.css | wc -l
130

```